In [1]:
import orbit_nb

# Neutrino Mass and Nature

In [2]:
%matplotlib inline
%reload_ext autoreload
%autoreload 2
import numpy as np
import matplotlib.pyplot as plt

import time
print(' Last version ', time.asctime() )

 Last version  Mon Apr 20 17:01:34 2026


**Objective:**

Oscillations prove $m_\nu \neq 0$, but only measure $\Delta m^2$. Three questions:

 * **How heavy are neutrinos?** — absolute mass scale
 * **How do they get mass?** — Dirac vs Majorana
 * **What is their nature?** — search for $\beta\beta0\nu$

### Roadmap

1. **Experimental limits on the neutrino mass** — oscillation recap, kinematic measurements (KATRIN, Project 8), cosmological bounds (Planck, DESI)
2. **How to provide mass to the neutrinos** — Dirac masses via Yukawa couplings, Majorana masses via the Weinberg operator, the seesaw mechanism
3. **How to determine the nature of the neutrino** — Dirac vs Majorana, the $\beta\beta0\nu$ process, effective Majorana mass
4. **The neutrinoless double beta decay search** — experimental techniques, current results (KamLAND-Zen, LEGEND, CUORE, NEXT), next-generation experiments
5. **Appendix: Heavy neutrinos** 
6. **References**


## 1. Experimental Limits on the Neutrino Mass

### Neutrino mass from oscillations: recap

Oscillations $\Rightarrow$ mass-squared differences:

$$\Delta m^2_{21} = (7.41^{+0.21}_{-0.20})\times 10^{-5}\,\mathrm{eV}^2$$

$$|\Delta m^2_{31}| = (2.511^{+0.028}_{-0.027})\times 10^{-3}\,\mathrm{eV}^2 \quad (\text{NuFit-6.0, 2024})$$

Splittings fixed, absolute scale $m_1$ unknown. Two orderings:

| Ordering | $m_1$ | $m_2$ | $m_3$ | $\sum m_i$ (min) |
|---|---|---|---|---|
| **NH** | lightest | $m_1+7\times10^{-3}$ eV | $m_1+50\times10^{-3}$ eV | $\geq 57$ meV |
| **IH** | $m_3+50\times10^{-3}$ eV | $m_3+51\times10^{-3}$ eV | lightest | $\geq 100$ meV |

**Ordering status:**

 * NuFit-6.0: NH preferred at $\sim 2.5\sigma$
 * JUNO first results (2025): NH at $\sim 2\sigma$ from reactors
 * Definitive measurement needs full JUNO / DUNE / HyperK

### Kinematic neutrino mass measurements

$\beta$-decay endpoint spectroscopy — **model-independent** probe of the effective electron-antineutrino mass:

$$m^2_\nu \equiv \sum_i |U_{ei}|^2\, m_i^2$$

 * Independent of Majorana/Dirac, CP phases, cosmology
 * Shape distortion maximal near the endpoint $E_0$:

$$
\frac{dN}{dT} \propto (E_0 - T) \sqrt{(E_0 - T)^2 - m^2_\nu} \;\Theta(E_0 - T - m_\nu)
$$

### KATRIN experiment

[KATRIN](http://www.katrin.kit.edu) (KIT, Karlsruhe) — precision spectroscopy of molecular tritium $\beta$-decay near endpoint $E_0 \approx 18.6$ keV:

$$\mathrm{T}_2 \to {}^3\mathrm{HeT}^+ + e^- + \bar{\nu}_e$$

| | | |
| :--: | :--: | :--: |
| <img src="./imgs/KATRIN_village.png" width=280> | <img src="./imgs/KATRIN_scheme.png" width=300> | <img src="./imgs/KATRIN_foto_inside.png" width=280> |
| KATRIN at KIT | 70 m beamline | Inside main spectrometer |

**Detection method** — Magnetic Adiabatic Collimation with Electrostatic filter (MAC-E):

 * Windowless gaseous tritium source: 100 GBq, 40 g/d throughput, ~99% purity
 * $\beta$'s magnetically guided to spectrometer
 * Main spectrometer (23 × 10 m): passes $E > qU$ at analyzing plane
 * Filter width $\mathcal{O}(1\,\mathrm{eV})$, pitch acceptance 51°, $B \lt 4.2$ T
 * Focal Plane Detector: 148-pixel Si-PIN array, ~95% efficiency
 * Calibration: $^{83m}$Kr + monoenergetic photoelectron sources

### KATRIN: first results (2019)

| | |
| :--: | :--: |
| <img src="./imgs/Katrin_spectrum.png" width=300> | <img src="./imgs/Katrin_E0m2plot.png" width=300> |
| Tritium $\beta$-decay near endpoint | $m^2_{\nu}$ fit across campaigns |

**KNM1** (2019, 35 d, 2M e$^-$) [arXiv:1909.06048](https://arxiv.org/abs/1909.06048):

$$m_\nu < 1.1 \;\mathrm{eV} \;(90\%\,\text{CL})$$

 * ~2× better than Mainz / Troitsk

**KNM1+2** (2021, 80 d, 6M e$^-$) [arXiv:2105.08533](https://arxiv.org/abs/2105.08533) — first sub-eV bound:

$$m_\nu < 0.8 \;\mathrm{eV} \;(90\%\,\text{CL})$$

### KATRIN: combined result (KNM1–KNM5)

Combined 5 campaigns, **259 days**, **36 M electrons** [K2]:

$$
m^2_\nu = -0.14^{+0.13}_{-0.15}\;\mathrm{eV}^2 \quad \Rightarrow \quad m_\nu < 0.45\;\mathrm{eV} \;(90\%\,\text{CL})
$$

 * Negative best-fit consistent with stat. fluctuation (p-value 0.84)
 * Upper limit: Lokhov-Tkachov; Feldman-Cousins $< 0.31$ eV
 * **Statistics-dominated** (0.108 eV²); leading syst: column density (0.052), energy-loss (0.034)
 * Two independent blinded analyses agree within 4%

**Outlook**: 1000 days by end of 2025 (~5× stats). Final sensitivity $\sim 0.3$ eV.

### KATRIN: evolution of results

| |
| :--: |
| <img src="./imgs/KATRIN_crono.png" width=300> |
| Evolution of the KATRIN $m_\nu$ upper limit |

| Campaign | Year | Data | Electrons | $m_\nu$ (90% CL) | Ref. |
|---|---|---|---|---|---|
| KNM1 | 2019 | 35 d | 2M | $< 1.1$ eV | [arXiv:1909.06048](https://arxiv.org/abs/1909.06048) |
| KNM1+2 | 2021 | 80 d | 6M | $< 0.8$ eV | [arXiv:2105.08533](https://arxiv.org/abs/2105.08533) |
| KNM1–5 | 2024 | 259 d | 36M | $< \mathbf{0.45}$ **eV** | [arXiv:2406.13516](https://arxiv.org/abs/2406.13516) |

Each step tightened systematics: shifted analyzing plane (KNM3), $^{83m}$Kr calibration, rear-wall cleaning (KNM5). Bound improved ~2× per campaign.

**Exercise: KATRIN — endpoint spectrum and kinematic neutrino mass**

KATRIN measures the tritium $\beta$-decay spectrum near the endpoint $Q_\beta = 18\,574$ eV.
Near the endpoint, the differential rate behaves as:

$$
\frac{dN}{dT} \propto (Q_\beta - T) \sqrt{(Q_\beta - T)^2 - m^2_{\nu_e}} \;\Theta(Q_\beta - T - m_{\nu_e})
$$

The effective mass $m^{eff}_{\nu_e} = \sqrt{\sum_i |U_{ei}|^2 m_i^2}$ sets the kinematic endpoint.

**Questions:**

1. Visualise the spectrum and Kurie plot for $m_\nu = 0$, 0.2, 0.45, 1 eV.
   Where does the difference become visible?
2. Why is measuring a sub-eV neutrino mass so challenging?
   Estimate the fraction of the spectrum affected for $m_\nu = 0.45$ eV.
3. KATRIN gives $m^{eff}_{\nu_e} < 0.45$ eV. Can we directly compare this with
   $m_{\beta\beta}$ from KamLAND-Zen? What are the key differences?

### Future experiments

### Project 8

[Project 8](https://www.project8.org) uses **Cyclotron Radiation Emission Spectroscopy (CRES)** to measure tritium $\beta$-decay with single-electron sensitivity.

- Demonstrated 18 eV resolution on a single electron in a magnetic trap
- Target: $m_{\nu_e} < 40$ meV with an atomic tritium source
- Complementary to KATRIN: different systematic uncertainties
- Timeline: atomic source R&D ~2026; physics run >2030

### Cosmological bounds on neutrino mass

Massive $\nu$ suppress small-scale structure (free-streaming) $\Rightarrow$ constraint on $\Sigma m_\nu = m_1 + m_2 + m_3$.

#### CMB + BAO: Planck 2018

Planck + BOSS + 6dFGS + SDSS [[P18]](https://arxiv.org/abs/1807.06209):

$$\Sigma m_\nu < 0.12 \text{ eV} \quad (95\%\,\mathrm{CL})$$

 * Most stringent laboratory-independent bound
 * Disfavours IH quasi-degenerate ($\Sigma m_\nu \gtrsim 0.10$ eV)
 * Compatible with both orderings in non-degenerate limit

#### DESI 2024: Baryon Acoustic Oscillations

The **Dark Energy Spectroscopic Instrument (DESI)** measured BAO from >6 million extragalactic objects (galaxies, quasars, Ly$\alpha$ forest) across $0.1 < z < 4.2$ in its first year of observations [[D24]](https://arxiv.org/abs/2404.03002).

Combined with CMB (Planck + ACT lensing), in flat $\Lambda$CDM + $\sum m_\nu$:

$$\Sigma m_\nu < 0.072 \text{ eV} \quad (95\%\,\mathrm{CL}) \quad [\text{DESI BAO + CMB}]$$

| | |
| :--: | :--: |
| <img src="./imgs/DESI_numass_2404.03002.png" width=600> |


**Tension!**

 * IH minimum from oscillations $\Sigma m_\nu^{\min}(\mathrm{IH}) \approx 100$ meV — above cosmological cap
 * NH minimum $\approx 59$ meV, close to cap

But the bound is **strongly model-dependent**.

A **direct, model-independent measurement** (KATRIN, Project 8) is decisive:

 * Only a lab measurement can settle the mass scale
 * Discrepancy would flag a problem in the cosmological model

### Summary: neutrino mass constraints

| Observable | Quantity | Best bound | Method |
|---|---|---|---|
| $\Delta m^2_{21}$ | mass splitting | $7.41\times10^{-5}$ eV² | Oscillations |
| $\Delta m^2_{31}$ | mass splitting | $2.51\times10^{-3}$ eV² | Oscillations |
| $m^{\mathrm{eff}}_{\nu_e}$ | endpoint mass | $< 0.45$ eV (90% CL) | KATRIN 2024 |
| $\Sigma m_\nu$ | sum of masses | $< 0.12$ eV (95% CL) | Planck 2018 |
| $\Sigma m_\nu$ | sum of masses | $< 0.072$ eV (95% CL) | DESI 2024 |
| $m_{\beta\beta}$ | Majorana eff. mass | $< 28$–$122$ meV (90% CL) | KamLAND-Zen 800 |

Three complementary probes: oscillations (splittings), kinematic + cosmology (absolute scale), $\beta\beta0\nu$ (Majorana nature).

## 2. How to Provide Mass to the Neutrinos

### Dirac neutrino masses

Dirac spinor — Dirac equation, decomposable into chiral parts:

$$\Psi = \Psi_L + \Psi_R$$

 * Global phase $\Psi' = e^{i\theta}\Psi$ $\Rightarrow$ conserved charge (electric, lepton number, …)
 * Mass term couples both chiralities:

$$-m \bar{\Psi} \Psi = -m (\bar{\Psi}_L \Psi_R + \bar{\Psi}_R \Psi_L)$$

**A mass term requires both left and right components.**

### Dirac masses in the SM

 * $L$ chiral fields: $SU(2)_L$ doublets; $R$ fields: singlets
 * Bare masses forbidden by gauge invariance $\Rightarrow$ arise via Yukawa + SSB

For one lepton family:

$$
L = \begin{pmatrix} \nu_L \\ l_L \end{pmatrix}, \quad l_R, \quad
\Phi = \begin{pmatrix} \phi^+ \\ \phi^0 \end{pmatrix}
\xrightarrow{\text{SSB}}
\langle \Phi \rangle = \begin{pmatrix} 0 \\ v/\sqrt{2} \end{pmatrix}
$$

Charged lepton: $\lambda_e \bar{L} \Phi \, e_R \Rightarrow m_e = \lambda_e \, v/\sqrt{2}$.

**Neutrinos remain massless** — no $\nu_R$ in the SM.

Need a **sterile right-chiral field** $\nu_R$, SM singlet, coupled via $\tilde{\Phi}$:

$$\tilde{\Phi} = i\sigma^2 \Phi^* \xrightarrow{\text{SSB}}
\langle \tilde{\Phi} \rangle = \begin{pmatrix} v/\sqrt{2} \\ 0 \end{pmatrix}$$

New Yukawa:

$$-\mathcal{L}_{\mathrm{mass},\,\nu} = \lambda_\nu \, \bar{L} \, \tilde{\Phi} \, \nu_R + \mathrm{h.c.} \quad\Rightarrow\quad m_\nu = \lambda_\nu \, v/\sqrt{2}$$

**Problem:** why is $m_\nu$ six orders of magnitude smaller than charged leptons?

$$\lambda_\nu \ll \lambda_l$$

No SM explanation — motivates the **Majorana** alternative.

### Majorana fermions

[E. Majorana](https://en.wikipedia.org/wiki/Ettore_Majorana) (1930s): neutral fermions can be **self-conjugate**.

| |
| :--: |
| <img src="./imgs/numass_majorana_abstract.png" width = 400> |

Majorana basis — all $\gamma^\mu$ purely imaginary:

$$
\gamma^0 = \begin{pmatrix} 0 & \sigma^2 \\ \sigma^2 & 0\end{pmatrix}, \;
\gamma^1 = \begin{pmatrix} i\sigma^3 & 0 \\ 0 & i\sigma^3\end{pmatrix}, \;
\gamma^2 = \begin{pmatrix} 0 & -\sigma^2 \\ \sigma^2 & 0\end{pmatrix}, \;
\gamma^3 = \begin{pmatrix} i\sigma^1 & 0 \\ 0 & i\sigma^1 \end{pmatrix}
$$

Dirac equation is real $\Rightarrow$ admits $\Psi(x) = \Psi^*(x)$: **particle = antiparticle**.

#### The Majorana condition

$$\Psi = \Psi^c, \quad \Psi^c \equiv C \bar{\Psi}^T = C \gamma^0 \Psi^*$$

$C$ obtained from covariance of the Dirac equation under charge conjugation.

In the Weyl-chiral basis:

$$
\Psi = \begin{pmatrix} \psi_L \\ \psi_R \end{pmatrix} \to
\Psi^c = i\gamma^2 \Psi^* = \begin{pmatrix} i\sigma^2 \psi_R^* \\ -i\sigma^2 \psi_L^* \end{pmatrix}
$$

Charge conjugate of a L spinor is R:

$$
\Psi_L = \begin{pmatrix} \psi_L \\ 0 \end{pmatrix} \to
(\Psi_L)^c = \begin{pmatrix} 0 \\ -i \sigma^2 \psi^*_L \end{pmatrix}
$$

#### Constructing the Majorana spinor

From a single L Weyl spinor $\psi_L$:

$$
\Psi^M = \Psi + \Psi^c = \begin{pmatrix} \psi_L \\ -i \sigma^2 \psi^*_L \end{pmatrix}
$$

**Properties of a Majorana fermion:**

 * Has L **and** R chiral components $\Rightarrow$ **has mass** (otherwise reduces to Weyl)
 * **No global phase symmetry** ($\psi_L \to e^{i\phi}\psi_L$ mixes signs in upper/lower blocks) $\Rightarrow$ **no conserved charge**
 * Purely neutral fermion

Neutrinos have mass — as demonstrated by oscillations — and they are the only fundamental fermions that could be Majorana particles.

*What, then, is the nature of neutrinos: Dirac or Majorana?*

### The Weinberg operator

Majorana mass for $\nu_L$:

$$-\frac{m}{2} (\overline{\nu_L} \nu^c_L + \overline{\nu^c_L} \nu_L)$$

Gauge invariance $\Rightarrow$ double Higgs coupling. The unique dim-5 operator [Wein]:

$$\mathcal{L}_W = -\frac{\alpha}{\Lambda} (\bar{L}\tilde{\Phi})(\tilde{\Phi}^T L^c) + \mathrm{h.c.}$$

After SSB:

$$m_\nu = \frac{\alpha v^2}{\Lambda}$$

For $\alpha \sim 1$, $v = 246$ GeV, $m_\nu \sim 1$ eV $\Rightarrow$ $\Lambda \sim 10^{14}$ GeV.

The Weinberg operator is the **only 5-dimensional operator** in the SM effective field theory expansion — it is the first signal of physics beyond the SM at low energy.

| | |
| :--: | :--: |
|<img src="./imgs/majo_weinberg_feynman.png" width = 300>| <img src="./imgs/majo_seesaw_feynman.png" width = 300>|
| Weinberg operator diagram| A realisation of the seesaw model |

Does this diagram remind you of something?

### Mixing and the $U_{PMNS}$ Matrix

With three families, the Yukawa couplings become $3\times3$ complex matrices.

The coupling with the Higgs allows Nature to relate flavour eigenstates to mass eigenstates via a unitary matrix $U$.

This is the matrix responsible, among other things, for neutrino oscillations.

The derivation depends on the neutrino nature: whether it is a Dirac or a Majorana fermion.

### Dirac masses: three families

Yukawa mixes flavours:

$$
-\mathcal{L}_H = \sum_{i,j=1}^3 \left(\lambda^l_{ij} \bar{L}'_i \Phi \, l'_{jR} + \lambda^\nu_{ij} \bar{L}'_i \tilde{\Phi} \, \nu'_{jR}\right) + \mathrm{h.c.}
$$

After SSB, diagonalise by two unitary transformations:

$$\frac{v}{\sqrt{2}} \lambda_\alpha = U^\dagger_\alpha \, M_\alpha V_\alpha$$

Weak ↔ mass eigenstates: $l_L = U_l l'_L$, $\nu_L = U_\nu \nu'_L$.

 * NC **unaffected** (unitarity)
 * CC acquires the **PMNS matrix**:

$$-\mathcal{L}_{CC} = \frac{g}{\sqrt{2}} \bar{l}_L \gamma^\mu (U^\dagger_l U_\nu) \nu_L W^+_\mu + \mathrm{h.c.}$$

Quark analogue: $V_{CKM}$.

### The $U_{PMNS}$ matrix (Pontecorvo–Maki–Nakagawa–Sakata) [MNS, Pon]

$$
U_{PMNS} = U^\dagger_l \, U_\nu
$$

The flavour and mass eigenstates are related by: $\nu'_\alpha = \sum_k U_{\alpha k} \, \nu_k$

**Dirac case**: $U_{PMNS}$ has 3 angles and **1 CP phase** ($\delta_{CP}$). We can reabsorb 5 phases using the global symmetries of the 6 Dirac fields.

| |
| :--: |
|<img src="./imgs/Umatrix_dirac.png" width = 600>|


### Majorana masses: three families

In the Majorana case, we extend the SM with the Weinberg operator for three families:

$$
-\mathcal{L}_m = \sum_{\beta, \beta'} \frac{\alpha^\nu_{\beta\beta'}}{\Lambda} (\bar{L}'_\beta \tilde{\Phi})(\tilde{\Phi}^T L'^c_{\beta'}) + \mathrm{h.c.}
$$

After SSB, the $3\times3$ symmetric complex matrix $\frac{v}{\sqrt{2}\Lambda}\alpha_\nu$ is diagonalised by a single unitary transformation.

The resulting mass matrix is symmetric — a hallmark of the Majorana nature.


**Majorana case**: Majorana fields have no global phase freedom, so only 3 phases (from charged leptons) can be reabsorbed. The $U_{PMNS}$ matrix has 3 angles and **3 phases**: $\delta_{CP}$ and two Majorana phases $\eta_1, \eta_2$.

| |
| :--: |
|<img src="./imgs/Umatrix_majorana.png" width = 600>|

The Majorana phases $\eta_{1,2}$ have no effect on oscillation probabilities — they can only be probed through lepton-number-violating processes such as $\beta\beta0\nu$.

### The seesaw mechanism

Weinberg operator is low-energy effective. UV completion at tree level $\Rightarrow$ **seesaw**.

| Type | New particle | Mass scale | Key feature |
|---|---|---|---|
| **Type I** | Heavy RH neutrino $N_R$ | $M_R \sim 10^{14}$ GeV | Minimal, most studied |
| **Type II** | Scalar triplet $\Delta$ | $M_\Delta \sim 10^{13}$ GeV | Same-sign dileptons |
| **Type III** | Fermion triplet $\Sigma$ | $M_\Sigma \sim 10^{13}$ GeV | Collider-testable |

#### Type I seesaw

Extend SM with $n$ heavy RH (sterile) neutrinos $N_{Ri}$, SM singlets.

$$-\mathcal{L} = y_\nu\,\bar{L}\,\tilde{\Phi}\,N_R + \frac{1}{2}M_R\,\overline{N^c_R}\,N_R + \text{h.c.}$$

After SSB ($m_D = y_\nu v/\sqrt{2}$, $M_R \gg m_D$):

$$
\mathcal{M} = \begin{pmatrix} 0 & m_D \\ m_D & M_R \end{pmatrix}
$$

Diagonalisation:

$$m_\nu \approx \frac{m_D^2}{M_R} \;(\text{light}), \qquad M_N \approx M_R \;(\text{heavy})$$

Larger $M_R$ $\Rightarrow$ smaller $m_\nu$ — hence *seesaw*.

 * $N_R$ can drive matter-antimatter asymmetry via **leptogenesis**
 * *New physics?* $\to$ see HNL Appendix

**Exercise: The seesaw mechanism and the PMNS matrix**

**Part A — Seesaw numerics:**

The Type I seesaw gives $m_\nu \approx m_D^2 / M_R$, where $m_D = y_\nu v/\sqrt{2}$.

1. Compute $m_\nu$ for $m_D = m_\tau = 1.777$ GeV and $M_R = 10^{14}$ GeV. Is the result in the right ballpark?
2. What value of $M_R$ would give $m_\nu = 0.05$ eV (the atmospheric scale) if $m_D = m_t = 173$ GeV?
3. The three families have very different $m_D$ (think $m_e, m_\mu, m_\tau$ or $m_u, m_c, m_t$). Does the seesaw naturally predict a neutrino mass hierarchy? Discuss.

**Part B — PMNS phases:**

4. A $3\times3$ unitary matrix has 9 real parameters (3 angles + 6 phases). How many phases are physical in the Dirac case? And in the Majorana case? Explain which field redefinitions are available in each case.
5. Why do the Majorana phases $\eta_1, \eta_2$ not affect oscillation probabilities? Show it explicitly for the two-flavour case.

## 3. How to Determine the Nature of the Neutrino

### Signature of Dirac and Majorana neutrinos

| |
| :--: |
| <img src="./imgs/pidecay_majorana_dirac.png" width = 600> |

**How to distinguish Majorana vs Dirac?** If Majorana, a $\nu$ from $\pi^+$ could produce either $\mu^+$ or $\mu^-$ — LNV.

**SM massless $\nu$: triple degeneracy**

|  |  |  |  |
| :--:  | :--: | :--: | :--: |
| $\nu$ | L chirality | $(-)$ helicity | $L_\nu = +1$ |
| $\bar{\nu}$ | R chirality | $(+)$ helicity | $L_\nu = -1$ |

Chirality, helicity and $L_\nu$ all equivalent. With mass, degeneracy breaks.

If Majorana, LNV amplitude is helicity-suppressed. Probabilities:

| | $h = -1$ | $h = +1$ |
| :--:  | :--: | :--: |
| $\nu_L$ | $\sim 1$ | $\mathcal{O}(m/E)$ |
| $\nu_R$ | $\mathcal{O}(m/E)$ | $\sim 1$ |

**Devastating suppression:**

 * $m_\nu \sim 1$ eV, $E \sim 1$ MeV $\Rightarrow$ $\sim 10^{-12}$
 * $m_\nu \sim 1$ eV, $E \sim 1$ GeV $\Rightarrow$ $\sim 10^{-18}$

LNV direct observation in scattering is impossible. Majorana and Dirac are *practically indistinguishable* — except via the rare $\beta\beta0\nu$.

### Double beta decay without neutrinos

| |
| :--: |
| <img src="./imgs/bb0nu_Feynman.png" width = 400> |

Only experimental probe of the Majorana nature:

$$(A, Z) \to (A, Z+2) + 2e^- + Q_{\beta\beta}$$

All three $\nu$ participate, each suppressed by $U^2_{ei} \, m_i / E$. Amplitude $\propto$ **effective Majorana mass**:

$$m_{\beta\beta} \equiv \left|\sum_i U^2_{ei} \, m_i\right|$$

### $\beta\beta 2\nu$: the allowed double beta decay

Proposed by [M. Goeppert-Mayer](https://en.wikipedia.org/wiki/Maria_Goeppert-Mayer) (1935):

$$(A, Z) \to (A, Z+ 2) + 2 e^- + 2 \bar{\nu}_e + Q_{\beta\beta}$$

 * 2nd-order weak decay — occurs in 35 isotopes where single $\beta$ is forbidden
 * Relevant isotopes: $^{48}$Ca, $^{76}$Ge, $^{82}$Se, $^{100}$Mo, $^{130}$Te, $^{136}$Xe
 * $T_{1/2} \sim \mathcal{O}(10^{18}$–$10^{24})$ yr

| |
| :--: |
|<img src="./imgs/bb2nu_isotopes_T12.png" width = 500 > |

Furry (1939) proposed the hypothetical $\beta\beta0\nu$ if Majorana:

$$\left(T^{0\nu}_{1/2}\right)^{-1} = G^{0\nu} \, \left| M^{0\nu} \right|^2 \, m^2_{\beta\beta}$$

with $G^{0\nu}$ the phase factor and $|M^{0\nu}|^2$ the nuclear matrix element (NME).

### Phase factor and nuclear matrix elements

|  |  |
| :--: | :--: |
|<img src="./imgs/bb0nu_table_G0nu_Qbb.png" width = 350>| <img src="./imgs/bb0nu_isotope_table_Qbb.png" width = 350>|

 * $G^{0\nu}/m^2_e \sim \mathcal{O}(10^{-26})$ $1/(\text{y}\,\text{eV}^2)$
 * Best isotope: largest phase space × $Q$ × natural abundance
 * No single winner — current experiments: $^{136}$Xe and $^{76}$Ge

| |
| :--: |
| <img src="./imgs/bb0nu_NME.png" width = 400> |

**NME uncertainty is large**: $M^{0\nu} \sim \mathcal{O}(1)$. Shell model, QRPA, IBM, ab initio $\Rightarrow$ factors 2–3 spread. Major theoretical challenge.

### The effective Majorana mass

Depends on $U_{\alpha i}$, lightest mass $m_0$, Majorana phases.

**NH:**

$$
m_{\beta\beta} = \left| m_0 c^2_{12} c^2_{13} + \sqrt{m^2_0 + \Delta m^2_{21}} \, s^2_{12} c^2_{13} e^{2i (\eta_2 -\eta_1)} + \sqrt{m^2_0 + \Delta m^2_{32} + \Delta m^2_{21}} \, s^2_{13} e^{-2i(\delta_{CP} + \eta_1)}\right|
$$

**IH:**

$$
m_{\beta\beta} = \left| m_0 s^2_{13} + \sqrt{|\Delta m^2_{32}| + m^2_0} \, c^2_{13} \left| s^2_{12} + c^2_{12} \, e^{2i(\eta_2 - \eta_1)} \right| \right|
$$

For $m_0 \to 0$: $m_{\beta\beta} \simeq 1.1$–$4.2$ meV (NH), $15$–$50$ meV (IH).

| |
| :--: |
| <img src="./imgs/mbb_vs_mlight.png" width = 400> |

Next-gen $\beta\beta0\nu$ target: cover the IH region.

### Interplay: $m^{eff}_{\nu_e}$, $m_{\beta\beta}$, and $\Sigma m_\nu$

| Quantity | Formula | Probes |
| :-- | :-- | :-- |
| $m^{eff}_{\nu_e}$ | $\sqrt{\sum_i |U_{ei}|^2 m_i^2}$ | absolute scale (Dirac or Majorana) |
| $m_{\beta\beta}$ | $|\sum_i U^2_{ei} m_i|$ | Majorana nature + CP phases |
| $\Sigma m_\nu$ | $m_1 + m_2 + m_3$ | cosmological (model-dependent) |

KATRIN $m^{eff}_{\nu_e} < 0.45$ eV constrains $m_{\beta\beta}$:

 * **Quasi-degenerate** ($m_0 \gg \Delta m_{ij}$): $m_{\beta\beta} \lesssim m^{eff}_{\nu_e} < 0.45$ eV
 * **IH**: $m_{\beta\beta} \sim 15$–$50$ meV, well below KATRIN $\Rightarrow$ $\beta\beta0\nu$ decisive

**Exercise: Effective Majorana mass spectrum**

The effective Majorana mass depends on the neutrino mass ordering, the lightest mass $m_{\rm min}$,
and the two unknown Majorana CP phases $\alpha_{21}, \alpha_{31} \in [0, 2\pi]$:

$$
m_{\beta\beta} = \left| c^2_{12}\,c^2_{13}\,m_1
               + s^2_{12}\,c^2_{13}\,m_2\,e^{i\alpha_{21}}
               + s^2_{13}\,m_3\,e^{i\alpha_{31}} \right|
$$

**Questions:**

1. What is the minimum value of $m_{\beta\beta}$ for NH when $m_1 \to 0$? And for IH when $m_3 \to 0$?
2. For which values of the Majorana phases does the NH band reach its minimum (cancellation)?
3. In IH, does $m_{\beta\beta}$ ever cancel to zero? Why or why not?

**Exercise: Half-life limits and NME uncertainty**

The connection between $T^{0\nu}_{1/2}$ and $m_{\beta\beta}$ goes through the nuclear matrix element $M'$
and the phase-space factor $G^{0\nu}$:

$$
\frac{1}{T^{0\nu}_{1/2}} = G^{0\nu}\,g_A^4\,|M'|^2\,\frac{m_{\beta\beta}^2}{m_e^2}
$$

**Questions:**

1. Compute $m_{\beta\beta}$ from the current limits on $T^{0\nu}_{1/2}$ for $^{136}$Xe (KamLAND-Zen 800),
   $^{76}$Ge (LEGEND-200 combined) and $^{130}$Te (CUORE). Use central, minimum and maximum NME values.
2. Which isotope currently gives the most stringent constraint on $m_{\beta\beta}$?
3. How much does the NME uncertainty affect the comparison between isotopes?

### Experimental signature of $\beta\beta0\nu$

| |
| :--: |
|<img src="./imgs/bb2nu_bb0nu_spectrum.png" width = 400> |

 * Signal: **peak** at $Q_{\beta\beta}$ (both $e^-$ carry all the energy)
 * $\beta\beta2\nu$: continuous spectrum (missing energy in neutrinos)

Expected events:

$$N_{\beta\beta} = \ln(2) \, \frac{N_A}{W} \, \frac{a \, \epsilon \, M \, t}{T^{0\nu}_{1/2}}$$

with $N_A$ Avogadro, $W$ molar mass, $a$ abundance, $\epsilon$ RoI efficiency, $M$ target mass, $t$ live time.

### Background sources and mitigation

**Signal identification:**

 * **Energy resolution** — defines RoI around $Q_{\beta\beta}$, $\Delta E = \mathrm{FWHM} \simeq 2.355\,\sigma$
 * **Topology** (2-$\beta$ tracks), pulse-shape analysis, fiducialisation

**Main backgrounds:**

 * **Cosmogenic muons + spallation** — mitigated by depth + active vetoes, reduction $\mathcal{O}(10^6)$
 * **Natural radioactivity** ($^{238}$U, $^{232}$Th chains) — $^{208}$Tl, $^{214}$Bi $\gamma$'s near $Q_{\beta\beta}$
 * Ultra-radiopure materials required: $\lesssim 10^{-6}$ Bq/kg

| | |
| :--: | :--: |
| <img src="./imgs/Th232_chain.png" width = 300 align='left'> | <img src="./imgs/U238_chain.png" width = 300 align='right'> |
| $^{232}$Th decay chain | $^{238}$U decay chain |


| |
| :--: |
| <img src="./imgs/muflux_undergroundlabs.png" width = 400 align='center'> |
| Muon flux vs depth (in metres water equivalent) for the main underground laboratories |

### Sensitivity regimes

Background events in RoI: $N_{\mathrm{bkg}} \propto b \, \Delta E \, M \, t$.

**Two regimes** for the half-life sensitivity $\mathcal{S}(T^{0\nu}_{1/2})$:

 * **Background-free** ($N_{\mathrm{bkg}} \ll 1$):

$$\mathcal{S}(T^{0\nu}_{1/2}) \propto \epsilon \, M \, t$$

 * **Background-dominated** ($N_{\mathrm{bkg}} \gg 1$):

$$\mathcal{S}(T^{0\nu}_{1/2}) \propto \sqrt{\frac{M \, t}{b \, \Delta E}}$$

| |
| :--: |
| <img src="./imgs/Xe_mbb_vs_exposure_bkginRoI.png" width = 400 align='center'> |

Projected $m_{\beta\beta}$ limits for an ideal $^{136}$Xe experiment vs exposure, for different BI in the RoI [[4]](https://arxiv.org/abs/1502.00581).

Target for IH coverage: tonne-scale + $\lesssim 1$ count/(RoI·ton·yr) — the holy grail.

### Detector requirements

The main ingredients for a successful $\beta\beta0\nu$ detector are:

   - **Large exposure** of the target isotope: $M \cdot t \sim \mathcal{O}(1)$ ton yr
   - **Excellent energy resolution**: $\Delta E \sim \mathcal{O}(1\text{--}0.1)$% FWHM at $Q_{\beta\beta}$
   - **Ultra-low background**: $b \sim \mathcal{O}(0.1)$ counts/(RoI ton yr), requiring radiopure materials, underground location, and active background rejection

Different isotopes and detection techniques offer complementary trade-offs between these three requirements, as we shall see in the next section.

## The Experimental Search for $\beta\beta0\nu$ Decays

### Liquid xenon experiments

$^{136}$Xe ($Q_{\beta\beta} = 2458$ keV): large homogeneous target, scalable to tonne.

**Two techniques:**

 * **Liquid Xe TPC** — charge + light readout — EXO-200
 * **Xe in liquid scintillator** — KamLAND-Zen

**Why $^{136}$Xe:**

 * High $Q_{\beta\beta}$, above most $\gamma$ backgrounds
 * 8.9% natural abundance, enrichable $> 80\%$
 * Self-shielding, scalable to tonnes

**Key backgrounds:**

 * $^{214}$Bi ($2448$ keV $\gamma$) — 10 keV below $Q_{\beta\beta}$, most dangerous
 * $^{208}$Tl ($2615$ keV $\gamma$) — Compton tail in RoI
 * $\beta\beta2\nu$ tail (limited by resolution)
 * Cosmogenic $^{137}$Xe ($T_{1/2} = 3.8$ min)
 * Radon emanation

### EXO-200 (2011–2015)

 * [WIPP](https://www.wipp.energy.gov), New Mexico (1600 m.w.e.)
 * 175 kg of $^{136}$Xe enriched to 80%
 * Symmetric LXe TPC: wires for ionization, APDs for scintillation
 * HFE-7000 bath, lead castle, active $\mu$ veto

| | |
| :--: | :--: |
| <img src="./imgs/EXO200_detector.png" width = 350 align='center'>  | <img src="./imgs/EXO200_photo.png" width = 450 align='center'> |
| EXO-200 drawing | EXO-200 in clean room |

Charge–light anti-correlation improves resolution; enables single-site/multi-site discrimination against $\gamma$'s.

| | |
| :--: | :--: |
| <img src="./imgs/EXO200_detector_operation.png" width = 300 align='center'> | <img src="./imgs/EXO200_calibration.png" width = 300 align='center'> |
| Detection technique | Calibration |

**EXO-200 results:**

- Total exposure: 234.1 kg·yr
- $^{136}\mathrm{Xe} \to ^{136}\mathrm{Ba} + 2e^-$, $Q_{\beta\beta} = 2458$ keV
- Energy resolution: 3% FWHM (1.2% $\sigma$ in phase-I, 1.15% $\sigma$ in phase-II)
- Background index: $1.7\times 10^{-3}$ cts/(keV kg yr) in phase-I; $1.9\times 10^{-3}$ cts/(keV kg yr) in phase-II
- Dominant backgrounds: $^{232}$Th chain ($^{208}$Tl) and $^{238}$U chain ($^{214}$Bi) from detector materials; multi-site event rejection reduced $\gamma$ backgrounds by a factor ~3

| |
| :--: |
| <img src="./imgs/EXO200_energy_spectrum.png" width = 700 align='center'> |
| Combined energy spectrum of EXO-200 (phases I + II) |

With 234.1 kg·yr exposure, EXO-200 established $T^{0\nu}_{1/2} > 3.5 \times 10^{25}$ yr at 90% CL [[5]](https://arxiv.org/abs/1906.02723).

#### nEXO (very likely out of the race!)

nEXO is a proposed 5-tonne liquid $^{136}$Xe TPC at SNOLAB, the successor to EXO-200.

  * SiPM light sensors, improved radiopurity (electroformed Cu), $\sigma_E = 0.8\%$ at $Q_{\beta\beta}$
  * Target: $T^{0\nu}_{1/2} > 10^{28}$ yr

| |
| :--: |
| <img src="./imgs/nEXO_fig.png" width = 500 align='center'> |

#### XLZD

Next-gen liquid-xenon TPC — merger of LZ + XENON + DARWIN.
Dual-purpose: **dark matter + $0\nu\beta\beta$** [[XLZD-DB]](https://arxiv.org/abs/2410.17137).

 * **60–80 t of natural Xe** (8.9% $^{136}$Xe $\Rightarrow$ ~5.3–7.1 t of $^{136}$Xe — no enrichment)
 * Dual-phase LXe TPC with SiPM readout, $\sigma_E \sim 0.8\%$ at $Q_{\beta\beta}$
 * Host lab under selection (SURF / LNGS / Boulby / SNOLAB)
 * Status (2025): **Design Book published**; first data early 2030s, 10-yr run

**$0\nu\beta\beta$ sensitivity (10 yr)** [[XLZD-bb]](https://arxiv.org/abs/2410.19016):

 * $3\sigma$ discovery: $T^{0\nu}_{1/2} > 5.7 \times 10^{27}$ yr
 * 90% CL exclusion: $T^{0\nu}_{1/2} > 1.3 \times 10^{28}$ yr
 * $m_{\beta\beta}$: 4.8–20.5 meV (exclusion), 7.3–31.3 meV ($3\sigma$)
 * **Excludes IH, starts probing NH** for most NMEs

### KamLAND-Zen (2011–2018)

 * Kamioka mine (2700 m.w.e.) — reuses KamLAND detector
 * $^{136}$Xe dissolved in LS, inside a 3 m nylon balloon at the center
 * Base mass: 300 kg of $^{136}$Xe; exposure 504 kg·yr

| |
| :--: |
| <img src="./imgs/KamLAND-zen_detector.png" width = 400 align='center'> |

 * Energy from total scintillation light; position from light pattern
 * $\Delta E \sim 10\%$ FWHM at $Q_{\beta\beta}$ — compensated by large mass
 * Dominant backgrounds: $^{110m}$Ag (cosmogenic on balloon), $^{214}$Bi, $\beta\beta2\nu$ tail
 * 1 m fiducial cut rejects balloon-originated events

| | |
| :--: | :--: |
| <img src="./imgs/KamLAND-zen_energy_spectrum.png" width = 350 align='center'>  | <img src="./imgs/KamLAND-zen_energy_spectrum_zoom.png" width = 350 align='center'>  |
| Energy spectrum (full range) | Zoom near $Q_{\beta\beta}$ |

With 504 kg·yr, KamLAND-Zen established $T^{0\nu}_{1/2} > 1.07 \times 10^{26}$ yr (90% CL) [[6]](https://arxiv.org/abs/1605.02889). Despite the modest energy resolution, the large mass and the absence of events in the RoI made this the world-leading limit at the time.

### KamLAND-Zen 800 (2019–2024)

Major upgrade of KamLAND-Zen with significantly improved radiopurity:

  * 745 kg of $^{136}$Xe (2× the previous phase)
  * 2× larger balloon with cleaner nylon film — reduced $^{214}$Bi contamination
  * Improved LS purification — reduced $^{110m}$Ag background
  * Total exposure: 2097 kg·yr (data taking ended January 2024)

| | |
| :--: | :--: |
| <img src="./imgs/KamLANDZEN800_Edistribution.png" width = 400 align='center'>  | <img src="./imgs/KamLANDZEN800_bkg.png" width = 300 align='center'>  |
| Energy spectrum | Background estimation in the RoI |

| |
| :--: |
| <img src="./imgs/KamLANDZEN800_mbb.png" width = 300 align='center'>  |
| Exclusion on $m_{\beta\beta}$ vs lightest neutrino mass |

Combined with previous KamLAND-Zen results [[8]](https://arxiv.org/abs/2406.11438):

$$
T^{0\nu}_{1/2} > 3.8 \times 10^{26} \; \text{yr at 90\% CL}
$$

$$m_{\beta\beta} < 28\text{--}122 \; \mathrm{meV}$$

The range reflects the nuclear matrix element uncertainty. This is currently the **most stringent limit** on the effective Majorana mass from any single isotope.

#### KamLAND2-Zen

KamLAND2-Zen is the planned next phase at Kamioka, with > 1 ton of $^{136}$Xe in an upgraded liquid scintillator.

Key upgrades: new LAB+PPO mixture with ~5× higher light yield; improved energy resolution to ~3% FWHM at $Q_{\beta\beta}$.

| |
| :--: |
| <img src="./imgs/KamLAND2_fig.png" width = 200 align='center'> |

### Germanium semiconductor experiments

High-purity germanium detectors enriched in $^{76}$Ge ($Q_{\beta\beta} = 2039$ keV) offer **excellent energy resolution** (~0.2% FWHM) and pulse-shape discrimination. The detector is simultaneously source and detector.

### GERDA (2015–2020)

- 37 High Purity Ge diodes (87% $^{76}\mathrm{Ge}$)
- Located in strings inside a 63 m³ LAr cryostat at LNGS
- $Q_{\beta\beta} = 2039$ keV, $^{76}\mathrm{Ge} \to ^{76}\mathrm{Se} + 2e^-$
- **Excellent energy resolution**: 0.2% FWHM
- Background index: $5.6 \times 10^{-4}$ c/(kg yr keV) for BEGe detectors

| | |
| :--: | :--: |
|<img src="./imgs/GERDA_detector.png" width = 250 align='left'> | <img src="./imgs/GERDA_detector_strings.png" width = 250 align='right'> |
| GERDA drawing | GERDA veto + Ge detectors |

| |
| :--: |
| <img src="./imgs/GERDA_energy_spectrum_phaseII.png" width = 600 align='center'> |

With 82.4 kg·yr, GERDA established $T^{0\nu}_{1/2} > 9 \times 10^{25}$ yr at 90% CL [[7b]](https://arxiv.org/abs/1909.02726).

### LEGEND-200

LEGEND-200 is the upgrade of GERDA (and the merger with the MAJORANA collaboration).

 * 200 kg of enriched $^{76}$Ge (×5 yr target), in GERDA cryostat at LNGS
 * Physics data since March 2023; Run 1 exposure: 61.0 kg·yr (golden dataset)
 * Background: $b = 0.5^{+0.3}_{-0.2} \times 10^{-3}$ cts/(keV kg yr) — world's lowest for $0\nu\beta\beta$

| |
| :--: |
| <img src="./imgs/LEGEND_cuts_drawing.png" width = 500 align='center'> |
| <img src="./imgs/LEGEND_pulseshape.png" width = 500 align='center'> |

| |
| :--: |
| <img src="./imgs/LEGEND_Espectrum_cuts.png" width = 500 align='center'> |


| | |
| :--: | :--: |
| <img src="./imgs/LEGEND200_spectrum_61ky_2505.10440.png" width = 500 align='center'> | <img src="./imgs/LEGEND200_zoom_61kry_2505.10440.png" width = 300 align='center'> |


Published results (PRL 2026, [arXiv:2505.10440](https://arxiv.org/abs/2505.10440)) [16]:
 * 1 event in RoI (1.5$\sigma$), no evidence of $0\nu\beta\beta$ signal
 * LEGEND-200 alone: $T^{0\nu}_{1/2} > 0.5 \times 10^{26}$ yr (90% CL)
 * Combined analysis (LEGEND-200 + GERDA + MAJORANA Demonstrator): $T^{0\nu}_{1/2} > 1.9 \times 10^{26}$ yr (90% CL), with median sensitivity $2.8 \times 10^{26}$ yr
 * Effective Majorana mass: $m_{\beta\beta} < 75$–$200$ meV (depending on NME)

#### LEGEND-1000

LEGEND-1000 is the ton-scale $^{76}$Ge experiment, successor to LEGEND-200, at **SNOLAB** (2 km underground).

 * 1000 kg of enriched $^{76}$Ge (enrichment $>90$%), in 4 modules of ~250 kg
 * Target: $3\sigma$ discovery sensitivity $T^{0\nu}_{1/2} > 10^{28}$ yr in 10 yr of live time
 * Covers the full inverted-ordering band: $m_{\beta\beta} \sim 9$–$21$ meV
 * First data expected by the end of the decade [[L1K26]](https://arxiv.org/abs/2501.10046)

| |
| :--: |
| <img src="./imgs/Legend_fig.png" width = 400 align='center'> |

### CUORE (2017–2024)

Cryogenic bolometric search for $0\nu\beta\beta$ of $^{130}$Te at LNGS.

 * 988 TeO$_2$ crystals (19 towers), ~206 kg of $^{130}$Te, operated at **~10 mK**
 * Natural Te (34% $^{130}$Te) — no enrichment
 * $Q_{\beta\beta} = 2527$ keV, $\Delta E \sim 7$ keV FWHM
 * BI $\sim 1.4 \times 10^{-2}$ cts/(keV·kg·yr) — **background-limited**

**Final result — 2 tonne·yr** (Science 2024, [arXiv:2404.04453](https://arxiv.org/abs/2404.04453)) [10]:

 * $T^{0\nu}_{1/2}(^{130}\text{Te}) > 3.5 \times 10^{25}$ yr (90% CI, Bayesian)
 * $m_{\beta\beta} \lesssim 70$–$240$ meV
 * Sensitivity gain needs particle ID $\Rightarrow$ CUPID

| | |
| :--: | :--: |
| <img src="./imgs/CUORE_assembly.jpg" width = 300 align='center'> | <img src="./imgs/CUORE_img_Nature22.webp" width = 500 align='center'> | 

#### CUPID — CUORE upgrade

Scintillating Li$_2$$^{100}$MoO$_4$ bolometers + Ge light detectors in the CUORE cryostat at LNGS [[CUPID]](https://arxiv.org/abs/2203.08386).

 * 1596 Li$_2$MoO$_4$ crystals, $\geq 95\%$ enriched in $^{100}$Mo (~240 kg of $^{100}$Mo)
 * $Q_{\beta\beta}(^{100}\text{Mo}) = 3034$ keV — above most $\gamma$ backgrounds
 * Heat + light $\Rightarrow$ rejects $\alpha$ background by ~100×
 * Target BI $\sim 10^{-4}$ cts/(keV·kg·yr), $\Delta E \sim 5$ keV FWHM

**Sensitivity:**

 * 90% CL exclusion: $T^{0\nu}_{1/2} > 1.8 \times 10^{27}$ yr
 * $3\sigma$ discovery: $\sim 10^{27}$ yr
 * $m_{\beta\beta} \sim 9$–$15$ meV — covers full IH
 * Data-taking: end of the decade

### High-pressure gaseous xenon experiments

Gaseous $^{136}$Xe at high pressure (10–15 bar) combines excellent energy resolution ($<1$% FWHM) with a unique **topological signature**: the two $\beta$-electrons produce tracks with energy depositions (blobs) at both ends, providing powerful background rejection.

### NEXT experiment (2017–)

 * 100 kg $^{136}$Xe HP Gas TPC
 * Excellent resolution: $< 1\%$ FWHM at $Q_{\beta\beta} = 2.48$ MeV
 * Topological signature: 2-electron tracks with blobs at both ends
 * [Canfranc Laboratory](https://lsc-canfranc.es), Spain

| |
| :--: |
| <img src="./imgs/NEXT_soft.png" width = 400 align='center'> |

**Concept:**

 1. S1 scintillation $\Rightarrow$ $t_0$
 2. Drifted $e^-$ to anode (1 mm/1 $\mu$s)
 3. S2 electroluminescence $\Rightarrow$ energy plane + tracking plane

| |
| :--: |
| <img src="./imgs/NEXT100_xsection.png" width = 400 align='center'> |



| | |
| :--: | :--: |
| <img src="./imgs/NEXT_topology_tracks.png" width = 400 align='center'> |  <img src="./imgs/NEXT_blobs_discrimination.png" width = 250 align='center'> |


| | |
| :--: | :--: |
|<img src="./imgs/NEXT_Tbb0nu_vs_Expo.png" width = 300 align='left'>  | <img src="./imgs/NEXT_Tbb0nu_vs_BkgIndex.png" width = 300 align='right'> |

Background index $= 4 \times 10^{-4}$ counts/(keV kg yr) and Exposure = 275 kg·yr [NEXT-1].

#### NEXT-White detector

| | |
| :--: | :--: |
| <img src="./imgs/NEW_foto.png" width = 400 align='center'>  |  <img src="./imgs/NEW_in_platform.png" width = 300 align='center'> |
| NEXT-White detector | Hall-A LSC NEXT installation |

### NEXT-100 (2024–)

A 100 kg $^{136}$Xe High-Pressure TPC at the LSC. Demonstrator for a larger 1-ton detector; targets background measurement, improved topological signal, and energy resolution at scale.

 * Commissioning started May 2024; first xenon operation at 4 bar in October 2024 [[NEXT-7]](https://arxiv.org/abs/2511.01710)
 * Design pressure: 13.5 bar; sensor planes: PMTs (energy) + SiPMs (tracking)
 * Target: $T^{0\nu}_{1/2} > 10^{26}$ yr with 3 yr of data

| | |
| :--: | :--: |
| <img src="./imgs/NEXT100_view_2505.17848.png" width = 450 align='center'>  |  <img src="./imgs/NEXT100_trk_ene_planes_2505.17848.png" width = 300 align='center'> |
| NEXT100 detector | NEXT100 traking and energy planes |

| | |
| :--: | :--: |
| <img src="./imgs/NEXT100_event_2511.02467.png" width = 500 align='center'>  |  <img src="./imgs/NEXT100_kr_ee_events_2505.17848.png" width = 200 align='center'> |
| NEXT100 S2 (waveform, track) | NEXT100 kr (41.5 keV), double escape ($e^+e^-)$ 1.6 keV |


**First NEXT-100 results (Nov 2025):**

 * **Sub-percent energy resolution** [[NEXT-6]](https://arxiv.org/abs/2511.02467): $(0.90 \pm 0.02)\%$ FWHM at the $^{208}$Tl photopeak (2615 keV), extrapolating to $\sim 0.93\%$ at $Q_{\beta\beta}$ — meets the sub-1% design spec.
 * **Krypton-83m calibration** [[NEXT-8]](https://arxiv.org/abs/2511.01710): 4.37% FWHM at 41.5 keV (point-like); maps detector response and confirms stability. Extrapolation to $Q_{\beta\beta}$ projects $\sim 0.5\%$ FWHM, surpassing the design target.

#### NEXT-HD

NEXT-HD (High Definition) is the next-generation HPTPC at LSC, following NEXT-100.

  * ~1 tonne of $^{136}$Xe at 15 bar
  * High-granularity readout for enhanced topological discrimination
  * Target: $T^{0\nu}_{1/2} > 10^{27}$ yr

| | |
| :--: | :--: |
| <img src="./imgs/NEXT1ton_design.png" width = 350 align='center'> | <img src="./imgs/NEXT1ton_reach_vs_expon.png" width = 250 align='center'> |

### Summary: current limits and future prospects

Current state of the art in $0\nu\beta\beta$ searches and expected sensitivities of the next-generation experiments. The background index (BI) is given in the standard unit $\text{cts}/(\text{keV}\cdot\text{kg}\cdot\text{yr})$

**Current 90% CL limits (2024–2026):**

| Experiment | Isotope | $Q_{\beta\beta}$ (keV) | Exposure | BI $\left[\tfrac{\text{cts}}{\text{keV}\cdot\text{kg}\cdot\text{yr}}\right]$ | $T^{0\nu}_{1/2}$ (yr) | $m_{\beta\beta}$ (meV) | Ref. |
|:--|:--:|:--:|:--:|:--:|:--:|:--:|:--:|
| KamLAND-Zen 800 | $^{136}$Xe | 2458 | 2097 kg·yr | $\sim 10^{-4}$ | $> 3.8 \times 10^{26}$ | $< 28$–$122$ | [[8]](https://arxiv.org/abs/2406.11438) |
| LEGEND-200 + GERDA + MAJORANA | $^{76}$Ge | 2039 | ~190 kg·yr | $5 \times 10^{-4}$ | $> 1.9 \times 10^{26}$ | $< 75$–$200$ | [[16]](https://arxiv.org/abs/2505.10440) |
| CUORE | $^{130}$Te | 2527 | 2 tonne·yr | $1.4 \times 10^{-2}$ | $> 3.5 \times 10^{25}$ | $< 70$–$240$ | [[10]](https://arxiv.org/abs/2404.04453) |

**Key observations:**

 * KamLAND-Zen currently leads in $m_{\beta\beta}$ due to its very large $^{136}$Xe mass, despite modest energy resolution ($\sim 10\%$ FWHM).
 * LEGEND delivers the best half-life limit per unit exposure thanks to $\sim 0.1\%$ FWHM resolution and BI below $10^{-3}$.
 * CUORE is background-limited ($\text{BI} \sim 10^{-2}$); the discovery gain of CUPID comes from scintillation particle ID, reducing the $\alpha$ background by two orders of magnitude.
 * All next-generation experiments aim at $\text{BI} \lesssim 10^{-4}$ to approach the zero-background regime at their planned exposures.
 * The ranges in $m_{\beta\beta}$ reflect the spread of nuclear matrix elements; the isotope diversity is essential to break this degeneracy if a signal is found.

### Next-generation $\beta\beta0\nu$ experiments

| |
| :--: |
| <img src="./imgs/bb0nu_future_experiments.png" width = 600 align='center'> |


**Next-generation sensitivities (discovery / exclusion targets):**

| Experiment | Isotope | Mass | BI target $\left[\tfrac{\text{cts}}{\text{keV}\cdot\text{kg}\cdot\text{yr}}\right]$ | Target $T^{0\nu}_{1/2}$ (yr) | $m_{\beta\beta}$ reach (meV) | Ref. |
|:--|:--:|:--:|:--:|:--:|:--:|:--:|
| KamLAND2-Zen | $^{136}$Xe | > 1 ton | $\sim 10^{-5}$ | $\sim 2 \times 10^{27}$ | $\sim 20$ | [[7]](https://arxiv.org/abs/2203.02139) |
| LEGEND-1000 | $^{76}$Ge | 1000 kg | $\sim 10^{-5}$ | $> 10^{28}$ | $9$–$21$ | [[L1K26]](https://arxiv.org/abs/2501.10046) |
| CUPID | $^{100}$Mo | 240 kg | $\sim 10^{-4}$ | $> 1.8 \times 10^{27}$ | $9$–$15$ | [[CUPID]](https://arxiv.org/abs/2203.08386) |
| NEXT-HD | $^{136}$Xe | ~1 ton | $\sim 10^{-5}$ | $> 10^{27}$ | $\sim 10$–$30$ | [[NEXT-5]](https://arxiv.org/abs/2005.06467) |


### The 2026 ESPP Briefing Book on $0\nu\beta\beta$

CERN Physics Briefing Book (Nov 2025), input to the 2026 European Strategy update [[ESPP26]](https://arxiv.org/abs/2511.03883). Chapter 6, Section 6.5 on $0\nu\beta\beta$.

**Key messages:**

 * $0\nu\beta\beta$ is *"the most promising way to establish a new physics scale connected to neutrino masses"*
 * **Portfolio strategy endorsed**: multiple, complementary isotopes and technologies
 * Strong European leadership in **CUPID, LEGEND, NEXT, XLZD**
 * Next milestone: $T^{0\nu}_{1/2} \sim 10^{27}$ yr (full IH coverage)
 * Strongly encourages theoretical work on **nuclear matrix elements**
 * R&D priorities: Ba-tagging, high-$Q_{\beta\beta}$ crystals ($^{48}$Ca, $^{150}$Nd, $^{116}$Cd, $^{96}$Zr)

**ton-scale experiments listed:**

| Experiment | Isotope | $^{iso}$Mass (kg) |
|:--|:--:|:--:|
| LEGEND-1000 | $^{76}$Ge | 900 |
| CUPID | $^{100}$Mo | 240 |
| nEXO | $^{136}$Xe | 1800 |
| KamLAND2-Zen | $^{136}$Xe | 720 |
| NEXT-HD | $^{136}$Xe | 900 |
| XLZD | $^{136}$Xe (nat) | 5316 |
| AMoRE-II | $^{100}$Mo | 85 |
| SNO+ Phase-I | $^{130}$Te | — |

**Theory.** Strongly encouraged: improving **nuclear matrix elements** — essential to extract $m_{\beta\beta}$ and exploit isotope complementarity.

**R&D priorities:**

 * Ba-tagging (Ba$^{++}$ ion detection in Xe TPCs)
 * High-$Q_{\beta\beta}$ crystals: $^{48}$Ca, $^{150}$Nd, $^{116}$Cd, $^{96}$Zr
 * Scalable ultra-low-background technologies

**Bottom line.** ESPP 2026 places $0\nu\beta\beta$ among the highest-priority small-scale programmes, sustaining the isotope-diverse, multi-technology approach of **LEGEND, CUPID, NEXT, XLZD**.

| |
| :--: |
| <img src="./imgs/bb0nu_timeline.svg" width = 600 align='center'> |

**Exercise: Experimental sensitivity as a function of exposure**

The half-life sensitivity of a $\beta\beta_{0\nu}$ search depends on exposure $M \cdot t$,
background index (BI) and energy resolution $\Delta E$:

$$
T^{0\nu}_{1/2}{}^{\rm sens} \propto \begin{cases}
M \cdot t & \text{(zero-background)} \\
\sqrt{M \cdot t / (\mathrm{BI} \cdot \Delta E)} & \text{(background-dominated)}
\end{cases}
$$

**Questions:**

1. Compare the half-life reach of LEGEND-1000 vs nEXO vs NEXT-HD using their projected parameters.
2. At what exposure does each experiment transition from the zero-background to background-dominated regime?
3. Which parameter (mass, BI, or energy resolution) has the largest impact on the $m_{\beta\beta}$ reach?

## Conclusions

**Measuring $m_\nu$ is one of the most urgent tasks in fundamental physics.**

 * Oscillations: $\Sigma m_\nu^{\min}(\mathrm{NH}) \approx 59$ meV, $\approx 100$ meV (IH)
 * KATRIN: $m^{\mathrm{eff}}_{\nu_e} < 0.45$ eV (goal $< 0.2$ eV)
 * Cosmology (Planck + DESI): $\Sigma m_\nu < 0.072$–$0.12$ eV
 * **Tension** cosmology vs. oscillations $\Rightarrow$ motivates model-independent lab measurements

**Majorana nature is experimentally very hard.**

 * Needs BI $\lesssim 10^{-4}$ cts/(keV·kg·yr), tonne-scale isotopic mass, high resolution, radiopurity, efficiency
 * Current: $T^{0\nu}_{1/2} > 3.8 \times 10^{26}$ yr (KamLAND-Zen), $> 1.9 \times 10^{26}$ yr (LEGEND-200 + GERDA + MAJORANA)
 * Next gen (LEGEND-1000, CUPID, KamLAND2-Zen, NEXT-HD, XLZD, nEXO) **covers IH** ($m_{\beta\beta} \gtrsim 15$ meV)
 * **NH still out of reach** — requires even more ambitious facilities

**$m_\nu$ connects Higgs physics and new physics at high scale.**

 * SM needs new physics: Dirac (tiny Yukawa $\sim 10^{-12}$) or Majorana (Weinberg op. at $\Lambda \sim 10^{14}$ GeV)
 * Majorana $\Rightarrow$ seesaw + leptogenesis + baryon asymmetry
 * Forces us to re-examine the Higgs mechanism for neutrinos

**Neutrinos are a fundamental component of the Universe.**

 * C$\nu$B number density: $n_\nu \simeq 336$ cm$^{-3}$ — 2nd most abundant after photons ($n_\gamma \simeq 411$ cm$^{-3}$)
 * $\Omega_\nu h^2 = \Sigma m_\nu / 93.14\,\text{eV}$
 * Minimum ($\Sigma m_\nu \simeq 0.06$ eV): $\Omega_\nu \simeq 1.4 \times 10^{-3}$ ($\sim 0.14\%$)
 * Current cap: $\Omega_\nu < 3 \times 10^{-3}$ ($\sim 0.5$–$1\%$ of $\Omega_m$)
 * Key role in large-scale structure

Pinning down $m_\nu$ and its Dirac/Majorana nature is about **the origin of mass, the matter-antimatter asymmetry, and the structure of the Universe**.

## Appendix: About heavy neutral leptons

### Why heavy neutrinos?

Type-I seesaw: add $n \geq 2$ right-handed (sterile) $N_R$, SM singlets. Mass matrix

$$\mathcal{M}_\nu = \begin{pmatrix} 0 & m_D \\ m_D^{T} & M_R \end{pmatrix} \Rightarrow m_\nu \simeq -m_D M_R^{-1} m_D^{T}, \quad M_N \simeq M_R$$

Active-sterile mixing:

$$|U_{\alpha N}|^{2} \simeq m_\nu / M_N$$

**Why look for HNLs?**

 * Direct evidence of the **seesaw mechanism**
 * $\nu$MSM (keV–GeV): explains $m_\nu$ + **leptogenesis** + dark matter
 * **Long-lived** for $M_N \lesssim m_W$ $\Rightarrow$ displaced-vertex signature
 * If Majorana, contribute to $m_{\beta\beta}$ in $0\nu\beta\beta$

Parameter space: plane $(M_N, |U_{\alpha N}|^{2})$, spanning many decades — no single experiment covers it all.

### HNL searches at the LHC

ATLAS, CMS, LHCb in $W \to \ell N$, $Z \to \nu N$ and heavy-flavour decays [[PDG-HNL]](https://pdg.lbl.gov/2025/listings/rpp2025-list-heavy-neutral-leptons.pdf).

**Mass ranges:**

 * $M_N \lesssim m_B$: LHCb, $B \to X \ell N$, $N \to \ell \pi$ displaced in VELO [[LHCb-HNL]](https://arxiv.org/abs/2512.14551)
 * $m_B \lesssim M_N \lesssim m_W$: ATLAS/CMS, $W \to \ell N$, displaced vertices; CMS extends to muon system for long $c\tau$
 * $M_N \gtrsim m_W$: prompt same-sign $\ell^{\pm}\ell^{\pm} jj$ — LNV signature unique to Majorana

**Status (Run 2 + partial Run 3):**

 * $|U_{\alpha N}|^{2} \lesssim 10^{-6}$–$10^{-7}$ at $M_N \sim 5$–$30$ GeV
 * HL-LHC projection: gain ~1 order of magnitude
 * Complementary to beam dumps below $m_B$

### SHiP — CERN SPS Beam Dump Facility

Dedicated beam-dump for **feebly-interacting particles** at CERN [[SHiP25]](https://arxiv.org/abs/2504.06692).

**Concept:**

 * 400 GeV protons on thick target $\Rightarrow$ $\sim 10^{20}$ charm + beauty hadrons in 15 yr
 * $D, B \to \ell N \Rightarrow$ displaced 2-track vertex ($N \to \mu\pi$, $\mu\ell\nu$) in evacuated 50 m decay volume
 * Background-free thanks to upstream absorber + muon shield

**Reach:**

 * $M_N \sim 0.5$–$5$ GeV, $|U_{\alpha N}|^{2}$ down to $\sim 10^{-10}$
 * Covers large portion of $\nu$MSM cosmological band
 * Also probes dark photons, dark scalars, ALPs, LDM

**Status (2025):**

 * TDR green light **2024**; ECN3 / BDF construction endorsed
 * Funding applications in Italy, Switzerland, UK, etc.
 * First beam **2031**, 15-yr run

### Other long-lived particle experiments

Complementary LLP detectors at the LHC [[LLP-rev]](https://arxiv.org/abs/2512.13011):

| Experiment | Host | Status | Mass reach | Concept |
|:--|:--:|:--:|:--:|:--|
| **FASER / FASER$\nu$** | LHC IP1 (480 m) | Running (Run 3) | $\lesssim 1$ GeV | Forward decay volume along beam line |
| **FASER2** | HL-LHC FPF | Proposed | $\lesssim 5$ GeV | 10× larger, dedicated HL-LHC facility |
| **CODEX-b** [[CODEX]](https://arxiv.org/abs/2505.05952) | LHCb cavern | Proposed (demonstrator) | $\lesssim 10$ GeV | $10^3$ m$^3$ shielded box, transverse to beam |
| **MATHUSLA** | Surface above CMS | Proposed | $\lesssim 30$ GeV | $\sim 100\times100$ m$^2$ tracker, $c\tau \sim 100$ m–km |
| **ANUBIS** | Shaft above ATLAS | Proposed | $\lesssim 30$ GeV | Tracker in PX14 shaft — low-cost MATHUSLA alternative |

**Complementarity:**

 * Sub-GeV: SHiP, FASER
 * Few–tens GeV: CODEX-b, MATHUSLA, ANUBIS
 * Up to TeV: ATLAS/CMS prompt

If HNLs in the 0.1–10 GeV range are discovered **and** are Majorana, they contribute directly to $m_{\beta\beta}$ — linking collider searches with the tonne-scale $0\nu\beta\beta$ programme.

## References

[Majo] E. Majorana, [Nuovo Cimento 14 (1937) 171](https://doi.org/10.1007/BF02961314). English translation: [arXiv:1404.4943](https://arxiv.org/abs/1404.4943).

[Wein] S. Weinberg, "Baryon and lepton-nonconserving processes", [Phys. Rev. Lett. 43 (1979) 1566](https://doi.org/10.1103/PhysRevLett.43.1566).

[MNS] Z. Maki, M. Nakagawa, S. Sakata, "Remarks on the unified model of elementary particles", [Prog. Theor. Phys. 28 (1962) 870](https://doi.org/10.1143/PTP.28.870).

[Pon] B. Pontecorvo, "Neutrino experiments and the problem of conservation of leptonic charge", Zh. Eksp. Teor. Fiz. 53 (1967) 1717 [Sov. Phys. JETP 26 (1968) 984].

[1] M. Goeppert-Mayer, "Double beta-disintegration", Phys. Rev. 48 (1935) 512.

[2] W. H. Furry, "On transition probabilities in double beta-disintegration", Phys. Rev. 56 (1939) 1184.

[3] J. Engel and J. Menéndez, "Status and future of nuclear matrix elements for neutrinoless double-beta decay: a review", Rep. Prog. Phys. 80 (2017) 046301, [arXiv:1610.06548](https://arxiv.org/abs/1610.06548).

[4] J. J. Gómez-Cadenas et al., "Sense and sensitivity of double beta decay experiments", JCAP 06 (2015) 007, [arXiv:1502.00581](https://arxiv.org/abs/1502.00581).

[5] G. Anton et al. (EXO-200), Phys. Rev. Lett. 123 (2019) 161802, [arXiv:1906.02723](https://arxiv.org/abs/1906.02723).

[6] A. Gando et al. (KamLAND-Zen), Phys. Rev. Lett. 117 (2016) 082503, [arXiv:1605.02889](https://arxiv.org/abs/1605.02889).

[7] S. Abe et al. (KamLAND-Zen 800), Phys. Rev. Lett. 130 (2023) 051801, [arXiv:2203.02139](https://arxiv.org/abs/2203.02139).

[8] S. Abe et al. (KamLAND-Zen 800, combined), Phys. Rev. Lett. 135 (2025) 262501, [arXiv:2406.11438](https://arxiv.org/abs/2406.11438).

[7b] M. Agostini et al. (GERDA), Phys. Rev. Lett. 125 (2020) 252502, [arXiv:1909.02726](https://arxiv.org/abs/1909.02726).

[9] S. Andringa et al. (SNO+), Adv. High Energy Phys. 2016, 6194250 (2016), [arXiv:1508.05759](https://arxiv.org/abs/1508.05759).

[10] D. Q. Adams et al. (CUORE), "Constraints on Lepton Number Violation with the 2 tonne·yr CUORE Dataset", Science 386, 1090 (2024), [arXiv:2404.04453](https://arxiv.org/abs/2404.04453).

[CUPID] W. R. Armstrong et al. (CUPID), "CUPID pre-CDR", (2022), [arXiv:2203.08386](https://arxiv.org/abs/2203.08386).

[ESPP26] Physics Preparatory Group, "Physics Briefing Book: Input for the 2026 update of the European Strategy for Particle Physics", CERN (Nov 2025), [arXiv:2511.03883](https://arxiv.org/abs/2511.03883).

[XLZD-DB] XLZD Collaboration, "The XLZD Design Book: Towards the Next-Generation Liquid Xenon Observatory for Dark Matter and Neutrino Physics", EPJC 85 (2025) 1192, [arXiv:2410.17137](https://arxiv.org/abs/2410.17137).

[XLZD-bb] XLZD Collaboration, "Neutrinoless Double Beta Decay Sensitivity of the XLZD Rare Event Observatory", J. Phys. G 52 (2025) 045102, [arXiv:2410.19016](https://arxiv.org/abs/2410.19016).

[PDG-HNL] Particle Data Group, "Heavy Neutral Leptons, Searches for", Review of Particle Physics (2025), [pdg.lbl.gov/2025/listings/rpp2025-list-heavy-neutral-leptons.pdf](https://pdg.lbl.gov/2025/listings/rpp2025-list-heavy-neutral-leptons.pdf).

[SHiP25] SHiP Collaboration, "SHiP experiment at the SPS Beam Dump Facility", (April 2025), [arXiv:2504.06692](https://arxiv.org/abs/2504.06692).

[LHCb-HNL] LHCb Collaboration, "Search for heavy neutral leptons in B-meson decays", JHEP 03 (2026) 178, [arXiv:2512.14551](https://arxiv.org/abs/2512.14551).

[CODEX] G. Aielli et al. (CODEX-b), "CODEX-b: Opening New Windows to the Long-Lived Particle Frontier at the LHC", (May 2025), [arXiv:2505.05952](https://arxiv.org/abs/2505.05952).

[LLP-rev] "Updated sensitivities to heavy neutral leptons at the LHC far detectors and SHiP", (Dec 2025), [arXiv:2512.13011](https://arxiv.org/abs/2512.13011).

[K1] M. Aker et al. (KATRIN), Nature Phys. 18 (2022) 160, [arXiv:2105.08533](https://arxiv.org/abs/2105.08533).

[K2] M. Aker et al. (KATRIN), Phys. Rev. Lett. 133 (2024) 251801, [arXiv:2406.13516](https://arxiv.org/abs/2406.13516). Also Science 388 (2025) 863.

[P18] N. Aghanim et al. (Planck), A&A 641 (2020) A6, [arXiv:1807.06209](https://arxiv.org/abs/1807.06209).

[D24] DESI Collaboration, JCAP 02 (2025) 021, [arXiv:2404.03002](https://arxiv.org/abs/2404.03002).

[P8] A. Ashtari Esfahani et al. (Project 8), Phys. Rev. Lett. 131 (2023) 102502, [arXiv:2212.05048](https://arxiv.org/abs/2212.05048).

[NF6] I. Esteban et al. (NuFit-6.0), JHEP 12 (2024) 216, [arXiv:2410.05380](https://arxiv.org/abs/2410.05380).

[16] LEGEND Collaboration, "First Results on the Search for Lepton Number Violating Neutrinoless Double Beta Decay with the LEGEND-200 Experiment", Phys. Rev. Lett. (2026), [arXiv:2505.10440](https://arxiv.org/abs/2505.10440).

[L1K] N. Abgrall et al. (LEGEND), AIP Conf. Proc. 1894 (2017) 020027, [arXiv:1709.01980](https://arxiv.org/abs/1709.01980).

[L1K26] LEGEND Collaboration, "Neutrinoless double-beta decay search with the LEGEND experiment", (2025), [arXiv:2501.10046](https://arxiv.org/abs/2501.10046).

[NEXT-1] J. Martín-Albo et al. (NEXT), "Sensitivity of NEXT-100 to neutrinoless double beta decay", JHEP 05 (2016) 159, [arXiv:1511.09246](https://arxiv.org/abs/1511.09246).

[NEXT-2] F. Monrabal et al. (NEXT), "The NEXT White (NEW) detector", JHEP 10 (2019) 230, [arXiv:1804.01780](https://arxiv.org/abs/1804.01780).

[NEXT-5] C. Adams et al. (NEXT), "Sensitivity of a tonne-scale NEXT detector for neutrinoless double-beta decay searches", JHEP 08 (2021) 164, [arXiv:2005.06467](https://arxiv.org/abs/2005.06467).

[NEXT-6] NEXT Collaboration, "Demonstration of Sub-Percent Energy Resolution in the NEXT-100 Detector", (2025), [arXiv:2511.02467](https://arxiv.org/abs/2511.02467).

[NEXT-7] NEXT Collaboration, "The NEXT-100 Detector", (2025), [arXiv:2505.17848](https://arxiv.org/pdf/2505.17848)

[NEXT-8] NEXT Collaboration, "First results of the NEXT-100 detector using $^{83m}$Kr decays", (2025), [arXiv:2511.01710](https://arxiv.org/abs/2511.01710).

[FY86] M. Fukugita and T. Yanagida, "Baryogenesis without grand unification", Phys. Lett. B 174 (1986) 45.

---